docs/_plugins/include_example.rb

```diff
@@ -38,7 +38,15 @@ def render(context)
       code = File.open(@file).read.encode("UTF-8")
       code = select_lines(code)
  
-      Pygments.highlight(code, :lexer => @lang)
+      rendered_code = Pygments.highlight(code, :lexer => @lang)
+
+      spark_version = site.config['SPARK_VERSION_SHORT']
+      hint = "<div class=\"hint\">Find the example here: " \
+        "<a href=\"https://github.com/apache/spark/tree/" \
+        "v#{spark_version}/examples/src/main/#{clean_markup}\">" \
```

Comment:
Could you make a JIRA so we remember to fix the issue with links not working before a branch is cut? We don't have to do it in this PR.